docs/source/llms/llm-evaluate/notebooks/rag-evaluation-llama2.ipynb

```diff
@@ -0,0 +1,876 @@
+    "\n",
+    "qa = RetrievalQA.from_chain_type(\n",
+    "    llm=OpenAI(temperature=0),\n",
+    "    chain_type=\"stuff\",\n",
+    "    retriever=docsearch.as_retriever(),\n",
+    "    return_source_documents=True,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "fd70bcf6-7c44-44d3-9435-567b82611e1c",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "## Evaluate the RAG system using `mlflow.evaluate()`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "de1bc359-2e40-459c-bea4-bed35a117988",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "Create a simple function that runs each input through the RAG chain"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "667ec809-2bb5-4170-9937-6804386b41ec",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [],
+   "source": [
+    "def model(input_df):\n",
+    "    answer = []\n",
+    "    for index, row in input_df.iterrows():\n",
+    "        answer.append(qa(row[\"questions\"]))\n",
+    "\n",
+    "    return answer"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "d1064306-b7f3-4b3e-825c-4353d808f21d",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "Create an eval dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "a5481491-e4a9-42ea-8a3f-f527faffd04d",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [],
+   "source": [
+    "eval_df = pd.DataFrame(\n",
+    "    {\n",
+    "        \"questions\": [\n",
+    "            \"What is MLflow?\",\n",
+    "            \"How to run mlflow.evaluate()?\",\n",
+    "            \"How to log_table()?\",\n",
+    "            \"How to load_table()?\",\n",
+    "        ],\n",
+    "    }\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "9c3c8023-8feb-427a-b36d-34cd1853a5dc",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "Create a faithfulness metric using `databricks-llama2-70b-chat` as the judge"
```

Comment:
Should we link to the docs on how to create the endpoint for llmama2 model? Like the example of the config yaml or something else?

Reply:
this is the foundation model hosted on databricks. no setup needed here.

Reply:
Ahh got it. That is why the target set it databricks. 

Should we add a comment somewhere about this since this the OSS repo about this integration or something?

Reply:
added